cms/templates/js/course-outline.underscore

```diff
@@ -411,6 +411,11 @@ if (is_proctored_exam) {
                 <div class="status-message">
                     <span class="icon fa <%- statusMessages[i].iconClass %>" aria-hidden="true"></span>
                     <p class="status-message-copy"><%- statusMessages[i].text %></p>
+                    <% if (enableHideFromTOC && xblockInfo.isSequential()) { %>
+                        <button type="button" class="button btn-primary subsection-share-link-button">
+                            <span class="icon fa fa-link icon-share-link" style="color:#fff"></span><%- gettext('Share') %>
```

Comment:
why are we still using styles here?

Reply:
For some reason if you add that style in the corresponding `scss` file it doesn't take it, that's why it's here.

Reply:
how are you trying to add it? can you show us?

Reply:
[This](https://github.com/openedx/edx-platform/pull/34043/files#diff-4af80dd6b4ff7370480fbadac8f9ea62fd34bc065e901bd03e460fed8f4b4d94R661) is the class that styles the icon, it even already has the `color` property, but it is not being taken into account.